react-components/src/tools/messages.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    locales: "en-US",
+    messages: {
+        newToolRequestDialogHeading: "New Tool Request",
+        toolNameLabel: "What is the name of the tool?",
+        toolDescLabel: "Briefly describe the tool:",
+        toolSrcLinkLabel: "Enter link for your tool's source (GitHub, BitBucket, DockerHub etc...):",
+        toolVersionLabel: "What is your tool's version?",
+        toolDocumentationLabel: "Enter link for your tool's documentation:",
+        toolInstructionsLabel: "Enter any other instructions or relevant information:",
+        toolTestDataLabel: "Enter link for your tool's test data:",
```

Comment:
Maybe the above strings could say "Enter *a* link to..." or "Enter *the* link to...".  We could also rephrase to "Provide a link to..."